Front back integration

jlangy opened the pull request:
- Load data from backend, sort and filter posts on Product Main page
- To setup, pull down backend master (if fix/toggle like post pull request isn't merged yet, pull that branch to see a working example)
- Run the backed with npm run dev
- Run the front end, posts should load in. If you get CORS errors, try out adding a cors extension to your browser

linyh0207 commented on screens/ProductMainScreen.js:
```diff
@@ -10,17 +10,60 @@ import SortByMenu from '../components/DropDownMenu/SortByMenu';
+    const searchUri = `http://localhost:8000/posts?latitude=5.2&longitude=4.3&radius=${searchRadius}000${activeTags.map(
+      (tag) => `&tag=${tag}`
+    )}`;
+    const apiData = await fetch(searchUri);
```
I ran the backend on port 8000 and started the front end. The product main screen does not have any product post show up neither on my iphone nor android stimulator. I tried to console log the error.

` const apiData = await fetch(searchUri).catch((error) => {
      console.error(error);
    });`

See screenshots below: 

![Screen Shot 2020-06-24 at 8 44 31 PM](https://user-images.githubusercontent.com/38272570/85651088-e2b4cb80-b65b-11ea-88d1-c07d05829756.png)
![Screen Shot 2020-06-24 at 8 44 23 PM](https://user-images.githubusercontent.com/38272570/85651094-e5172580-b65b-11ea-9636-bf2766183b64.png)

jlangy commented:
Added some things:
- Fixed network issues, changed domain to 10.0.2.2. Should work for emulators now
-  Search bar filtering fixes, shows message when nothing matched, searches for start of tag matching only,
overflow wrap added when chips overflow, liking updating in local state now.
